{"name": "test_social_media_add[LinkedIn-https://www.linkedin.com/in/username]", "status": "failed", "statusDetails": {"message": "AssertionError: assert '• Sosyal medya adresiniz başarıyla eklendi.' in '• Bu sosyal medya zaten mevcut.'\n +  where '• Sosyal medya adresiniz başarıyla eklendi.' = const.social_media_add_message\n +  and   '• Bu sosyal medya zaten mevcut.' = <pages.page_social_media.PageSocialMedia object at 0x000001CFB720ECC0>.toast_message\n +    where <pages.page_social_media.PageSocialMedia object at 0x000001CFB720ECC0> = <test_social_media.TestSocialMedia object at 0x000001CFB7195790>.page", "trace": "self = <test_social_media.TestSocialMedia object at 0x000001CFB7195790>, media = 'LinkedIn', url = 'https://www.linkedin.com/in/username'\n\n    @pytest.mark.parametrize(\"media, url\", cr.read_json(\"../data/socialMedia.json\"))\n    def test_social_media_add(self, media, url):\n        self.prc_social_media(True)\n        if media == \"Instagram\":\n            self.page.save_button.click()\n            self.page.errors()\n            assert const.field_error_message in self.page.field_error\n        self.page.account_type.send_keys(media, Keys.ENTER)\n        self.page.account_url.send_keys(url)\n        self.page.save_button.click()\n        self.page.operationResult()\n        screenshot_dir = '../screenshots'\n        os.makedirs(screenshot_dir, exist_ok=True)\n        screenshot_path = os.path.join(screenshot_dir, f'{media}_added.png')\n        sleep(1)\n        self.driver.save_screenshot(screenshot_path)\n>       assert const.social_media_add_message in self.page.toast_message\nE       AssertionError: assert '• Sosyal medya adresiniz başarıyla eklendi.' in '• Bu sosyal medya zaten mevcut.'\nE        +  where '• Sosyal medya adresiniz başarıyla eklendi.' = const.social_media_add_message\nE        +  and   '• Bu sosyal medya zaten mevcut.' = <pages.page_social_media.PageSocialMedia object at 0x000001CFB720ECC0>.toast_message\nE        +    where <pages.page_social_media.PageSocialMedia object at 0x000001CFB720ECC0> = <test_social_media.TestSocialMedia object at 0x000001CFB7195790>.page\n\ntest_social_media.py:59: AssertionError"}, "parameters": [{"name": "media", "value": "'LinkedIn'"}, {"name": "url", "value": "'https://www.linkedin.com/in/username'"}], "start": 1717675639427, "stop": 1717675644308, "uuid": "e92dbee1-b3c2-433c-8ceb-1bfd1dea07f3", "historyId": "03983773a579e71c8c53c8cefd7d6be0", "testCaseId": "7033cac7b77f9d33cb43af19bde5dfa4", "fullName": "tests.test_social_media.TestSocialMedia#test_social_media_add", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_social_media"}, {"name": "subSuite", "value": "TestSocialMedia"}, {"name": "host", "value": "Ercan"}, {"name": "thread", "value": "13916-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_social_media"}]}